Mlem/App/Views/Shared/Toast/ToastView.swift

```diff
@@ -20,6 +20,9 @@ struct ToastView: View {
     // These symbols only have a single hierarchical layer, so we render it as `.secondary`
     static let dimmedSymbols: Set<Icon> = [.lemmy.block]
     
+    // These symbols need `.symbolVariant(.circle.fill)` applied to render properly
+    static let circledSymbols: Set<Icon> = [.general.success, .general.error, .general.failure]
```

Comment:
I think `.general.undo` needs to be in here too. It's the fallback icon (line 58) though I don't believe we're using it anywhere yet

```suggestion
    static let circledSymbols: Set<Icon> = [.general.success, .general.error, .general.failure, .general.undo]
```